kustomize/commands/internal/util/configmapSecretFlagsAndArgs_test.go

```diff
@@ -14,13 +14,13 @@ import (
 
 func TestDataValidation_NoName(t *testing.T) {
 	fa := ConfigMapSecretFlagsAndArgs{}
-	require.Error(t, fa.Validate([]string{}))
+	require.Error(t, fa.ValidateAdd([]string{}))
```

Comment:
It would be nice if tests for `ValidateSet` and `UpdateLiteralSources` were added.

Reply:
Thanks for flagging this!! I totally forgot 😅 added some tests, could you please take a look and check if they address your concerns?

Reply:
Thanks!